examples/example_modal_decomposition_data.py

```diff
@@ -0,0 +1,111 @@
+import numpy as np
+
+from lasy.profiles.transverse.transverse_profile_from_data import (
+    TransverseProfileFromData,
+)
+from lasy.profiles.transverse.hermite_gaussian_profile import (
+    HermiteGaussianTransverseProfile,
+)
+from lasy.profiles.longitudinal.gaussian_profile import GaussianLongitudinalProfile
+from lasy.profiles.combined_profile import CombinedLongitudinalTransverseProfile
+from lasy.utils.mode_decomposition import hermite_gauss_decomposition
+
+import matplotlib.pyplot as plt
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+import skimage
+
+
+# Define the transverse profile of the laser pulse
+img_url = "https://user-images.githubusercontent.com/27694869/228038930-d6ab03b1-a726-4b41-a378-5f4a83dc3778.png"
+intensityData = skimage.io.imread(img_url)
+rows, cols = intensityData.shape
+intensityData[intensityData < 2.1] = 0
+pixel_calib = 0.186e-6
+lo = (
+    -intensityData.shape[0] / 2 * pixel_calib,
+    -intensityData.shape[0] / 2 * pixel_calib,
+)
+hi = (
+    intensityData.shape[0] / 2 * pixel_calib,
+    intensityData.shape[0] / 2 * pixel_calib,
+)
```

Comment:
? Are we just fortunate that it's a square image?
```suggestion
lo = (
    -intensityData.shape[0] / 2 * pixel_calib,
    -intensityData.shape[1] / 2 * pixel_calib,
)
hi = (
    intensityData.shape[0] / 2 * pixel_calib,
    intensityData.shape[1] / 2 * pixel_calib,
)
```

Reply:
nope, that was a mistake